src/io.jl

```diff
@@ -50,12 +52,12 @@ function read_meshfile(filename::String)
     coordinates = coordinates[:, leaf_cells]
     levels = levels[leaf_cells]
 
-    return center_level_0, length_level_0, leaf_cells, coordinates, levels
+    return center_level_0, length_level_0, leaf_cells, coordinates, levels, ndims
   end
 end
 
 
-function read_datafile(filename::String)
+function read_datafile(filename::String, ndims::Int64)
```

Comment:
```suggestion
function read_datafile(filename)
```

I think instead of passing the dimensions, they should be read from the file.

Reply:
I am not sure how to do that. A simple `ndims = read(attrs(file)["ndims"])` works, but I don't know why in `read_meshfile` it's this whole block: 
```julia
if exists(attrs(file), "ndims")
  ndims = read(attrs(file)["ndims"])
else
  ndims = read(attrs(file)["ndim"]) # FIXME once Trixi's 3D branch is merged & released
end
```
Which one should I use here?

Reply:
The correct one is `ndims`, but keep it as it is to support legacy files. At some point (let's say, spring 2021), we should be OK in removing the `ndim` version.